cobbler/settings.py

```diff
@@ -208,9 +208,6 @@ def __init__(self):
         Constructor.
         """
         self._clear()
-        if (self.manage_tftp or self.manage_tftpd) and not os.path.isdir(self.tftpboot_location):
-            print("TFTP directory '{}' not found".format(self.tftpboot_location), file=sys.stderr)
-            sys.exit(1)
 
```

Comment:
Travis is failing now:

```
The command "pyflakes *.py cobbler/*.py cobbler/modules/*.py cobbler/web/*.py cobbler/web/templatetags/*.py bin/cobbler* bin/*.py web/cobbler.wsgi" exited with 1.
```

Reply:
I partially agree with that, but since the Python code change is part of fixing the source installation I thought it would be best to change it right here.

The linter is complaining about the following:

```shell
cobbler/settings.py:22:1 'sys' imported but unused
```